docs/source/_static/theme_overrides.css

```diff
@@ -61,22 +61,22 @@
   border: none;
 }
 
-.contib-card .sd-card-footer p.sd-card-text{
+.contrib-card .sd-card-footer p.sd-card-text{
```

Comment:
A change like this has impact on how things actually look, so should be evaluated (maybe best done separately). 

I know it's an obvious typo fix, but there is no guarantee this css actually works, because due to the typo we never properly evaluated it.

Reply:
I'll peel it into another PR...

Reply:
#39032